tox.ini

```diff
@@ -70,8 +73,8 @@ commands =
     bash {toxinidir}/.travis/runblack.sh --check --diff .
 
 [testenv:pylint]
-envdir = {toxworkdir}/env-{env:TRAVIS_PYTHON_VERSION:2.7}
-basepython = python2.7
+envdir = {toxworkdir}/env-{env:TRAVIS_PYTHON_VERSION:3.6}
+basepython = python3.6
```

Comment:
can't use python2.7?

Reply:
If we can install pylint>=2.5 with python2.7, we can... Or instead of creating symlinks, copying the custom module_utils, it might work with python2.7...

If just switching back to python2.7 in tox.ini, it fails like this:
```
************* Module certificate_request
E:279, 0: Unable to import 'ansible_collections.fedora.system_roles.plugins.module_utils.certificate.providers.__init__' (import-error)
************* Module certificate.providers.certmonger
E:  5, 0: Unable to import 'ansible_collections.fedora.system_roles.plugins.module_utils.certificate.providers.base' (import-error)
************* Module certificate.providers
E:  1, 0: Unable to import 'ansible_collections.fedora.system_roles.plugins.module_utils.certificate.providers.certmonger' (import-error)
ERROR: InvocationError for command /usr/bin/bash .travis/runpylint.sh --errors-only (exited with code 2)
```

Reply:
> instead of creating symlinks, copying the custom module_utils

It doesn't like symlinks?

Reply:
> > instead of creating symlinks, copying the custom module_utils
> 
> It doesn't like symlinks?

That's what i'm suspecting... Let me test copying with python2.7 if we prefer 2.7. :)

Reply:
It's not that I prefer python2.7, but 1) I don't want to change this arbitrarily 2) we need coverage on python2.7.  It would be great if we could do pylint (and pytest) on python 2 and python 3 (but we can do that later).

Reply:
One problem is we cannot use this pylint change (using python3 + pylint>=2.5) for network due to its referring basestring (See https://github.com/linux-system-roles/network/blob/main/module_utils/network_lsr/utils.py#L17). Even if its in the else of PY3, pylint fails by the undefined variable error. :(

Reply:
Why does it fail if there is `noqa:F821` ? And how is it possible that lint passes for the original code in the repository?